src/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicenseManager.cs

```diff
@@ -357,5 +357,12 @@ public static License Validate(Type type, object instance)
 
             return lic;
         }
+
+        /// <summary>
+        /// Retrieves the typehandle of the interop helper
+        /// </summary>
+        private static IntPtr GetLicenseInteropHelperType() {
```

Comment:
We should not have this method. It is useless indirection. The CoreCLR should be fixed to fetch LicenseInteropHelper directly.

Reply:
Also, it should be called out that my intent for COM activation is that it should be done in managed as much as possible - this code will be definitely be consumed from SPCL for managed COM server activation. See https://github.com/dotnet/coreclr/blob/ffc347acdbcaa05bc85fcd51bca57c551bc33955/src/System.Private.CoreLib/src/System/Runtime/InteropServices/ComActivator.cs#L240